src/smc_sagews/smc_sagews/sage_salvus.py

```diff
@@ -616,12 +616,13 @@ def interact_control(arg, value):
     c._opts['var'] = arg
     return c
 
-def sage_eval(x, locals=None):
-    x = str(x).strip()
-    if x.isspace():
-        return None
+def sage_eval(x, locals=None, **kwds):
+    if isinstance(x, str):
+        x = str(x).strip()
+        if x.isspace():
```

Comment:
I'm sorry, I don't get this test. If it is a space, isn't it then the empty string?

`" ".strip().isspace()` → `False`